addons/udes_stock/views/stock_op_analysis.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<odoo>
+  <data>
+    <!-- Move Line Report tree view -->
+    <record id="udes_stock.stock_op_analysis_tree" model="ir.ui.view">
+      <field name="name">udes_stock.stock.move.line.tree</field>
+      <field name="model">stock.move.line</field>
+      <field name="arch" type="xml">
+
+        <tree string="Move Line" create="0" decoration-muted="state == 'cancel'" decoration-danger="(state not in ('cancel','done'))">
+      	  <field name="id" invisible="1"/>
+          <field name="location_id"/>
+          <field name="u_done_by" />
+          <field name="u_done_datetime"/>
```

Comment:
This field comes from module udes_scanned_by_tracking and udes_stock is not dependent on that module. 
If someone uses the udes_stock module wouldn't this report throw error?

Reply:
Thanks, added dependency.